test/Makefile

```
@@ -104,6 +104,21 @@ smem_lockstep_parity_test: smem_lockstep_parity_test.o
 header_insert_test:header_insert_test.o
 	$(CXX) -o $@ $^ $(LIBS)
 
+packed_text_test: packed_text_test.o ../src/packed_text.o
+	$(CXX) -o $@ $^ $(LIBS)
+
+packed_text_test.o: ../src/packed_text.h
```

checkmake (0.3.2)

[warning] Target "packed_text_test.o" should be declared PHONY.

(phonydeclared)